game-runtimes/unreal/unreal.mdx

```diff
@@ -15,25 +15,22 @@ description: 'Unreal runtime for Rive.'
 
 The rive-unreal runtime uses the [Rive Renderer](https://rive.app/renderer) and supports the following:
 
-* Metal on Mac
-
-* Metal on iOS
-
-* D3D11 and D3D12 on Windows
-
-* OpenGL on Android (Note: be sure to uncheck "Support Vulkan" in Project Settings.)
+- Metal on Mac
+- Metal on iOS
+- Vulkan, D3D11, and D3D12 on Windows
+- OpenGL on Android (Note: be sure to uncheck "Support Vulkan" in Project Settings.)
 
 Planned support for:
 
-* Vulkan on Linux, Android, and Windows
+- Vulkan on Linux and Android.
 
 ## Bug Reports
 
 If you encounter any errors or unexpected crashes while integrating the Rive Unreal runtime, we recommend logging a detailed issue directly to the [rive-unreal](https://github.com/rive-app/rive-unreal/issues) repository.
 
 ## Feature Support
 
-The rive-unreal runtime uses the [Rive C++ runtime](https://github.com/rive-app/rive-cpp). For more details on runtime support, see the [Feature Support](/feature-support) page. Refer to the following table for what is currently supported in the Unreal runtime.
+The rive-unreal runtime uses the [Rive C\+\+ runtime](https://github.com/rive-app/rive-cpp). For more details on runtime support, see the [Feature Support](/feature-support) page. Refer to the following table for what is currently supported in the Unreal runtime.
 
 | **Feature**                                                      | **Supported** |
```

Comment:
Left a similar comment to @damzobridge for Unity, maybe we should be pointing to the Feature Support page instead.